lgr/logger.go

```diff
@@ -1,23 +1,52 @@
 package lgr
 
 import (
-	"log"
+	"fmt"
 
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
 // Logger represents logger.
 type Logger struct{ log *zap.SugaredLogger }
 
+// Allowed logger's levels.
+const (
+	lvlDebug = "DEBUG"
+	lvlInfo  = "INFO"
+	lvlError = "ERROR"
+)
+
 // New initialize logger
-func New() *Logger {
-	// TODO: Extend config and make adjustments
-	logger, err := zap.NewDevelopment()
+func New(logLevel string) (*Logger, error) {
+	if logLevel != lvlDebug && logLevel != lvlError && logLevel != lvlInfo {
```

Comment:
I think it's better to validate config when you getting it. And here we have to pass constants
```go
// Allowed logger's levels.
const (
	lvlDebug = "DEBUG"
	lvlInfo  = "INFO"
	lvlError = "ERROR"
)
```

Ont top of that, these constants should be public so any other package will be able to use right values.